src/storage/gc_worker.rs

```diff
@@ -746,9 +746,9 @@ impl<S: GCSafePointProvider, R: RegionInfoProvider> GCManager<S, R> {
     /// of `safe_point`. TiKV won't do any GC automatically until the first time `safe_point` was
     /// updated to a greater value than initial value.
     fn initialize(&mut self) -> GCManagerResult<()> {
-        info!("gc-manager is initializing");
+        debug!("gc-manager is initializing");
         self.safe_point = 0;
-        self.try_update_safe_point();
+        self.wait_for_next_safe_point()?;
```

Comment:
This line is changed recently in master and I guess you've changed it back by mistake.

Reply:
Oops, fixed.